core/gui/src/app/workspace/component/workflow-editor/context-menu/context-menu/context-menu.component.ts

```diff
@@ -25,13 +25,28 @@ export class ContextMenuComponent {
     this.registerWorkflowModifiableChangedHandler();
   }
 
-  public isSelectedOperatorValid(): boolean {
-    const highlightedOperatorIDs = this.workflowActionService.getJointGraphWrapper().getCurrentHighlightedOperatorIDs();
-    if (highlightedOperatorIDs.length !== 1 || !this.isWorkflowModifiable) {
+  public canExecuteOperator(): boolean {
+    if (!this.hasExactlyOneOperatorSelected() || !this.isWorkflowModifiable) {
```

Comment:
Do we need to handle the case of no operator being selected?

Reply:
It is logical to handle this scenario by ensuring that if the operator is not selected, execution cannot proceed. Also, this function can be invoked even when no operator is selected, so we need to account for and handle this case appropriately.